src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/LicenceSummary.kt

```diff
@@ -4,14 +4,14 @@ import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
 
-@Schema(description = "Response object for a created licence")
-data class CreateLicenceResponse(
+@Schema(description = "Response object which summarises a licence")
+data class LicenceSummary(
   @Schema(description = "Internal identifier for this licence generated within this service", example = "123344")
   val licenceId: Long,
 
   @Schema(description = "Licence type code", example = "AP")
   val licenceType: LicenceType,
 
-  @Schema(description = "The status of this new licence after creation", example = "IN_PROGRESS")
+  @Schema(description = "The status of this licence", example = "IN_PROGRESS")
   val licenceStatus: LicenceStatus,
```

Comment:
Might be useful to include a few more values here? e.g. crn, nomsId, pnc, forename, surname, dob? We probably don't need this yet though, so fine for now.